src/main.rs

```diff
@@ -47,11 +44,7 @@ async fn main() -> Result<(), Box<dyn Error>> {
         .await?
         .expect("Missing 'Parasite' document.");
     println!("Movie: {}", movie);
-    let title = movie
-        .get("title")
-        .expect("No title found")
-        .as_str()
-        .expect("title should have been a string!");
+    let title = movie.get_str("title")?;
```

Comment:
Instead of doing `get(...)` followed by `as_something`, `Document` has methods to get a value as a certain type (or return an error if it's not that type or not found). If you prefer the explicit error messages, this might not be preferable for this example, although in a production setting, this would be mitigated by just defining your own error type with a custom message.